extensions/parquet/table/src/test/java/io/deephaven/parquet/table/ParquetTableReadWriteTest.java

```diff
@@ -93,6 +93,7 @@ public final class ParquetTableReadWriteTest {
     private static final ParquetInstructions REFRESHING = ParquetInstructions.builder().setIsRefreshing(true).build();
 
     // TODO(deephaven-core#5064): Add support for local S3 testing
+    // The following tests are disabled by default, as they require a AWS access key and secret key to be set
```

Comment:
The reason why they should be disabled by default IMO is not b/c they require key / secret, but because it's a test against a _remote_ system.